setup.cfg

```diff
@@ -5,3 +5,6 @@ versionfile_source = distributed/_version.py
 versionfile_build = distributed/_version.py
 tag_prefix =
 parentdir_prefix = distributed-
+
+[bdist_wheel]
+universal=1
```

Comment:
> When I build I ususally do python setup.py install sdist bdist_wheel --universal

It's another way of specifying the same flag in a more permanent way, so you don't have to remember to pass the argument every time. The resulting wheel is the same if you've been doing `--universal` already.